From 6db1a2686f0f073438d36f5fa0f97b787842b0f2 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremyhu@freedesktop.org>
Date: Tue, 17 Jan 2012 23:55:23 -0800
Subject: [PATCH 2/8] Revert "Fix include order with Xdmcp on WIN32"

This reverts commit 0e9246def562be97cc816f824f702233a826ec56.

This change caused build failures because <X11/Xdmcp.h> was never
included under any circumstance.  This is because the check for
HASXDMAUTH was moved before the inclusion of config.h (via xcbint.h)
which defined it.

Found-by: Tinderbox
Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
Reviewed-by: Jon TURNEY <jon.turney@dronecode.org.uk>
---
 src/xcb_auth.c |    7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git libxcb-1.8/src/xcb_auth.c libxcb-1.8/src/xcb_auth.c
index ea4ad8f..21c26b8 100644
--- libxcb-1.8/src/xcb_auth.c
+++ libxcb-1.8/src/xcb_auth.c
@@ -36,10 +36,6 @@
 #include <rpc/types.h>
 #endif
 
-#ifdef HASXDMAUTH
-#include <X11/Xdmcp.h>
-#endif
-
 #ifdef _WIN32
 #include "xcb_windefs.h"
 #else
@@ -52,6 +48,9 @@
 #include "xcb.h"
 #include "xcbint.h"
 
+#ifdef HASXDMAUTH
+#include <X11/Xdmcp.h>
+#endif
 
 enum auth_protos {
 #ifdef HASXDMAUTH
-- 
1.7.9.2

